src/toolkits/object_detection/one_shot_object_detection/one_shot_object_detector.cpp

```diff
@@ -32,9 +32,11 @@ namespace one_shot_object_detection {
 namespace data_augmentation {
 
 flex_dict build_annotation(ParameterSampler &parameter_sampler,
-                           std::string label, size_t object_width,
-                           size_t object_height, long seed) {
-  parameter_sampler.sample(seed);
+                           std::string label, 
```

Comment:
can `label` be `const`?

Reply:
Typically this should either be just `std::string` if we want to pass by value and `const std::string&` if we want to pass by reference. See https://google.github.io/styleguide/cppguide.html#Use_of_const

> For a function parameter passed by value, const has no effect on the caller, thus is not recommended in function declarations.

Here I would recommend a const reference, since we don't modify the value

Reply:
Agreed. It should be a const reference.